org-code-javabuilder/theater/src/main/java/org/code/theater/Prompter.java

```diff
@@ -0,0 +1,8 @@
+package org.code.theater;
+
+public class Prompter {
+  protected Prompter() {}
+
+  // When implemented, should ultimately return an Image object.
+  public void getImage() {}
```

Comment:
Do you want to go ahead and have the method signature be the expected (`Image getImage(String)`) but just return `null`? That's typically how we've implemented no-op APIs in the past.

Reply:
Oops, hold on a sec. Looking back at spec, I may be misunderstanding what this method is supposed to do :).